src/components/editor/DraftSpec.tsx

```diff
@@ -46,7 +55,11 @@ function DraftSpecEditor() {
                 mutate()
                     .then(() => {})
                     .catch(() => {});
+
+                return updatePromise;
```

Comment:
Need to return this promise so that we know when the `onChange` is over and the editor can turn off the "saving..." message.